portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACUsage.path

```diff
@@ -0,0 +1,26 @@
+<html>
+<head>
+<title>ACUsage</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">ACUsage</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>USAGE_ADDONS_LIMITS</td>
+	<td>//h4[contains(.,'Add-ons')]/..//span[contains(@class,'limit-amount') and contains(.,'${key_limits}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>USAGE_ADDONS_PRICE</td>
+	<td>//h4[contains(.,'Add-ons')]/..//div[contains(@class,'annual-price') and contains(.,'${key_price}')]</td>
+	<td></td>
```

Comment:
Hey @AfonsoHSantos , is it possible to create additional paths or update these to be more precise?

With the current paths, we are only asserting if there is an element on the screen with that value, but we don't know where this value is or to what information the value is related... For example, imagine that 500 USD is wrongly assigned to Page Views addons price, instead of Individuals... the test will pass, since it will be able to find the info on the screen, however, it will be wrong since it will be related with the wrong place/info.

Thanks!

Reply:
fixed